.rubocop.yml

```diff
@@ -22,6 +28,18 @@ Style/ClassAndModuleChildren:
 Style/Documentation:
   Enabled: false
 
+# This forces us to create a new object for no real reason.
+Style/MultipleComparison:
+  Enabled: false
+
+# Doesn't feel right ...
+Style/NumericPredicate:
+  Enabled: false
+
+# TODO: remove this on Rails 5.
```

Comment:
Why only on Rails 5?

Reply:
Some methods are not there in 4.2 (e.g. `none?` as a class method: `Registry.none?`). This was an issue I had to manually fix. In Rails 5 I believe this can be removed.

Reply:
Gotcha. 👍